phantom_token.c

```diff
@@ -247,7 +246,7 @@ static ngx_int_t handler(ngx_http_request_t *request)
 
     u_char *bearer_token_pos;
 
-    if ((bearer_token_pos = (u_char *)strcasestr((char*)request->headers_in.authorization->value.data, BEARER)) == NULL)
+    if ((bearer_token_pos = (u_char *)ngx_strcasestrn((char*)request->headers_in.authorization->value.data, BEARER,BEARER_SIZE)) == NULL)
```

Comment:
The `u_char *` cast is unnecessary with that function, it seems. On Mac, I get an error if `BEARER` isn't cast to `char*` from `const char*`. Therefore, I suggest this: 

`if ((bearer_token_pos = ngx_strcasestrn(request->headers_in.authorization->value.data, (char*)BEARER, BEARER_SIZE)) == NULL)`